### ISSUE TITLE
Just a note from the gitter: attrs collection attribute setter does not pass back to the referenced variable that is passed from the block within the each_with_index

### ISSUE THREAD

**Issue by pencilcheck** [voltrb/volt#150]
If I have a _skills collection, each skill is a string, if I do
```
{{ _skills.each_with_index do |skill, index| }}
  <:editable-text value="{{ skill }}" />
{{ end }}
```

If I change skill the value will not be saved, but if I changed so I pass in the label of skill instead it works.
```
{{ _skills.each_with_index do |skill, index| }}
  <:editable-text value="{{ skill._label }}" />
{{ end }}
```
But that would mean skill is not a string anymore.

I wonder if there is a way to get this working without always resort into a workaround?

link:
https://github.com/voltrb/volt-editable-text/blob/master/app/editable-text/views/main/index.html

**Comment by ryanstout** [comment]
Fixed in 0.9.1.pre2 I believe, verify if you can to make sure.

**Comment by pencilcheck** [comment]
On 0.9.1.pre3 and it is still not working.
